src/js/rx/actions/prescriptions.js

```diff
@@ -21,7 +21,31 @@ export function loadData(id) {
     };
   }
 
-  return dispatch => fetch('/rx-api/prescriptions')
+  return dispatch => dispatch({ type: 'LOAD_PRESCRIPTION_FAILURE' });
+}
+
+export function loadPrescriptions(options) {
+  let uri = '/rx-api/prescriptions';
```

Comment:
Oh, I think that change would be in our local `config.proxy.js` files (`path: '/api/v0'`). We're using `/rx-api/[rest_of_uri]` for development, I think. Don't know if we keep it this way for production, but @alexose might know.